code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -68,6 +68,8 @@
 	if(..())
 		return 1
 
+	if(pulling && istype(pulling, /mob/living/carbon/human))
+		stop_pulling()

```

Comment:
Also, it doesn't actually fix it - mutual pulling will make the `return 1` occur before this even executes.